### DISCUSSION TITLE
Can I use PrimeTween in commercial projects?

### DISCUSSION THREAD

**Discussion by 0xdw** [KyryloKuzyk/PrimeTween#27] [FAQ] [answered]
Hey, there are a few questions I have. I highly appreciate it if you can answer those questions. Because I have migrated the whole game to this from DOTween to test it out. And it works pretty well.

1. Is this software free to use in commercial apps?
2. I have a few project-specific features to be implemented. With those features, this is going to be directly embedded in the project's binary itself. Does the current license permit such things?

**Comment by KyryloKuzyk** [comment, marked as answer]
Hey,

1. Yes. As long as PrimeTween is compiled inside the app's binary.
2. Yes. You can modify PrimeTween's source code in your own projects.

If you want more details for your legal team, PrimeTween is covered by the Asset Store [EULA](https://unity.com/legal/as-terms). In short, you're allowed to use and modify Asset Store assets in your own apps and games, but you're not allowed to create derivative assets based on the original assets.

**Comment by 0xdw** [comment]
Thanks for your quick answer. Yes, its going to be inside the game. I'll post any bug I found.

**Comment by KyryloKuzyk** [comment]
> Thanks for your quick answer. Yes, its going to be inside the game. I'll post any bug I found.

Thanks! Please also leave an honest review on Asset Store when you become more familiar with PrimeTween :)
